c19-backend/api/views.py

```diff
@@ -8,7 +8,22 @@
 class CovidScreenListView(APIView):
 
     def post(self, request, format=None):
-        # serializer = Covid19Serializer(data=request.data)
-        # if serializer.is_valid():
-        #     ehrbase.CONNECTION.post(data=serializer.data)
-        return Response(data={'fakeresponse': 'foo'})
+        if (patient := request.user.c19_api_patient_profile):
```

Comment:
if condition:
body
no
if (condition):
body

You can break the line by '' if condition is to long:
if a == True and
b == False